docs/reference-capabilities/capability-subtyping.md

```diff
@@ -1,38 +1,45 @@
+## Subtypes of unique capabilities
 
-## Ephemeral substitution
+When it comes to talking about unique capabilities, the situation is a bit more complex. With variables, we only had the six basic capabilities,
+but we're talking about expressions here. We will have to work with aliased and unaliased forms of the capabilities.
 
-The last case to consider is when we have an ephemeral reference capability. For example, if we have some `iso` and we `consume` it or do a destructive read, the type we get is `iso^`, not `iso`.
+From here, let's talk about ephemeral capabilities. Remember that the way to get an ephemeral capability is by _unaliasing_, that is, moving a value out of a
+named location with `consume` or destructive read.
 
-* `iso^ <: iso`. This is pretty simple. When we give an `iso^` a name, by assigning it to something or passing it as an argument to a method, it loses the `^` and becomes a plain old `iso`. We know we gave up our previous `iso`, so it's safe to have a new one.
-* `trn^ <: trn`. This works exactly like `iso^`. The guarantee is weaker (_write uniqueness_ instead of _read and write uniqueness_), but it works the same way.
-* `ref^ <: ref^` and `ref^ <: ref` and `ref <: ref^`. Here, we have another case. Not only is a `ref^` a subtype of a `ref`, it's also a subtype of a `ref^`. What's going on here? The reason is that an ephemeral reference capability is a way of saying "a reference capability that, when aliased, results in the base reference capability". Since a `ref` can be aliased as a `ref`, that means `ref` and `ref^` are completely interchangeable.
-* `val^`, `box^`, `tag^`. These all work the same way as `ref`, that is, they are interchangeable with the base reference capability. It's for the same reason: all of these reference capabilities can be aliased as themselves.
+Subtyping here is surprisingly simple: `iso^` is a sub-capability of absolutely everything, and `trn^` is a sub-capability of `ref` and `val`. Let's go through the interesting cases again with these two:
 
-__Why do `ref^`, `val^`, `box^`, and `tag^` exist if they are interchangeable with their base reference capabilities?__ It's for two reasons: __reference capability recovery__ and __generics__. We'll cover both of those later.
+* `iso^ <: trn^`. An `iso^` guarantees there's no readable or writable aliases, whereas `trn^` just needs no writable aliases.
+* `trn^ <: ref`. A `trn^` reference can be used to read and write, which is enough for `ref`.
+* `trn^ <: val`. A `trn^` reference has no writable aliases. A `val` requires global immutability, so we can forget our writable access in order to get `val`, since we know no other aliases can write.
+
+## Temporary unique access
+
+We just talked about stable aliases, and unaliasing, but what about when we just use a variable without making an alias?
```

Comment:
What's "stable" in this context (stable aliases)?

Reply:
A stable alias is a named reference to something. Not sure exactly what the step would be to improve it within this paragraph, but I definitely see that it needs to be changed.